SPEC.md

```diff
@@ -77,7 +77,7 @@ An `issue` represents a single instance of a real or potential code problem, det
 * `content` -- **Optional**. A markdown snippet describing the issue, including deeper explanations and links to other resources.
 * `categories` -- **Required**. At least one category indicating the nature of the issue being reported.
 * `location` -- **Required**. A `Location` object representing the place in the source code where the issue was discovered.
-* `other_locations` -- **Optional.** An array of `Location` objects useful for engines which highlight more than one source location in an issue.
+* `trace` -- **Optional.** A `Trace` object representing other interesting source code locations related to this issue.
```

Comment:
I thought about this too and I think we're fine, since we can update `codeclimate` and `builder` to handle both sorts of output.

It gets tricky the other direction, because it's hard for us to force updates to all engines. But this direction shouldn't be too much work.

(Though handling versioning would be nice)